src/app/dashboard/[subdomain]/settings/css/page-component.tsx

```diff
@@ -115,3 +117,90 @@ export default function SettingsCSSPage() {
     </SettingsLayout>
   )
 }
+
+const PreviewButton: FC<{
+  css: string
+  subdomain: string
+}> = ({ css, subdomain }) => {
+  const { t } = useTranslation("dashboard")
+
+  const datasetRef = useRef({
+    previewWindowOrigin: "",
+    previewWindow: null as null | Window,
+  }).current
+  const [isInPreview, setIsPreview] = useState(false)
+
+  const handlePreview = () => {
+    let url: URL
+
+    if (IS_DEV) {
+      url = new URL(`http://${subdomain}.localhost:2222/`)
+    } else {
+      url = new URL(location.href)
+      // TODO
```

Comment:
I want to know what customize domain or second-level domain name is, and replace this fake URL